src/layerelement.cpp

```diff
@@ -707,8 +708,12 @@ double LayerElement::GetAlignmentDuration(
             return duration->GetInterfaceAlignmentMensuralDuration(num, numbase, mensur);
         }
         if (this->Is(NC)) {
-            const Neume *neume = vrv_cast<const Neume *>(this->GetFirstAncestor(NEUME));
-            if (neume->IsLastInNeume(this)) {
+            const Object *neume = this->GetFirstAncestor(NEUME);
+            assert(neume);
+            const Object *syllable = neume->GetFirstAncestor(SYLLABLE);
+            assert(syllable);
+            // Add a gap after the last nc of the last neume in the syllable
+            if ((neume->GetLast() == this) && (syllable->GetLast() == neume)) {
                 return 256;
```

Comment:
Should these be defined as constants? NEUME_LARGE_SPACE vs NEUME_SMALL_SPACE, or something like that?